versions/build.ver

```diff
@@ -40,7 +40,6 @@ export w3nco_ver=2.4.1
 export wgrib2_ver=2.0.8_wmo
 export wrf_io_ver=1.2.0
 export zlib_ver=1.2.11
-
```

Comment:
Since there is no change to this file, please check out build.ver directly from the "dev-sci" branch.